phac-nml/irida-next
# Workflow Submission:  Change row components to column components for faster turbo updates

## What does this PR do and why?
_Describe in detail what your merge request does and why._

Refactor of the samplesheet component to allow turbo updates of only a single column of the table instead of updating n number of cells separately.

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

No real changes, now looks like:

**Light**
![image](https://github.com/phac-nml/irida-next/assets/11295750/3b8e2211-15a5-44ab-8717-40d6068053ec)

**Loading State**
![loading](https://github.com/phac-nml/irida-next/assets/11295750/c4e61fcf-efb8-4cb1-96ab-7c4ceee44688)

**Dark**
![image](https://github.com/phac-nml/irida-next/assets/11295750/ece7683f-7b20-4172-8e80-24b07057b97f)

## How to set up and validate locally
_Numbered steps to set up and validate the change are strongly suggested._

Try running a pipeline, there should be no changes.

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
